packages/web-config/tsconfig.json

```diff
@@ -56,7 +56,8 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "noErrorTruncation": true,
-    "experimentalDecorators": true
+    "experimentalDecorators": true,
+    "skipLibCheck": true
```

Comment:
`web-config` was the only package that didn't have skipLibCheck enabled. I attempted to update our node types to v16 but it seems to cause more problems than it fixed; https://github.com/cypress-io/cypress/issues/21329 was logged to dedicate more time this effort.

Reply:
`web-config` is used in several packages btw. What motivated this change?

Reply:
Bumping to electron 18 caused `yarn type-check` to fail for this package due to conflicts in some deep transient dependencies. I assume this was due to electron's bumping to @types/node@16: https://github.com/cypress-io/cypress/pull/21418/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR8720-R8723. Looks like we might have been inadvertently(?) resolving against v15 before?

My type-config-fu is admittedly weak. It seems like we can resolve the failures if I add an explicit @types/node@14 dependency to this package. Would that be preferred? My understanding was that this flag forced a check on types we are actually referencing rather than on all dependencies.

Reply:
Hmm, this seems fine after reading the config doc: https://www.typescriptlang.org/tsconfig#skipLibCheck I don't think you need to change it. The doc says we can use Yarn `resolutions` maybe to pin it to 14 everywhere, but again, I don't think it's really an issue.